app/models/hub_edos/student_api/v2/registrations.rb

```diff
@@ -3,7 +3,7 @@ module StudentApi
     module V2
       class Registrations < ::HubEdos::StudentApi::V2::Proxy
         def url
-          "#{@settings.base_url}/v2/students/#{@campus_solutions_id}?inc-regs=true"
+          "#{@settings.base_url}/v2/students/#{@campus_solutions_id}?inc-regs=true&as-of-date=#{1.year.from_now.strftime("%Y-%m-%d")}"
```

Comment:
@developish - Why are we adding this hard-coded to the proxy, instead of making it an optional thing? Do we really foresee no one needing to query the current registration states.... rather than 1 year from now?

Reply:
The API shows all registrations before the as-of-date, which is assumed to be the current date, so it only returns registrations up to today. This change will include current and future terms, more like previous versions of the API.

Reply:
Oooh, so this is ensuring that we're getting a fuller data set. Okay.